sdks/python/apache_beam/ml/transforms/tft.py

```diff
@@ -280,14 +271,6 @@ def __init__(
     self.elementwise = elementwise
     self.name = name
 
-  def get_artifacts(self, data: common_types.TensorType,
-                    col_name: str) -> Dict[str, common_types.TensorType]:
-    shape = [tf.shape(data)[0], 1]
-    return {
-        col_name + '_min': tf.broadcast_to(tft.min(data), shape),
-        col_name + '_max': tf.broadcast_to(tft.max(data), shape)
-    }
-
```

Comment:
Can we rip this out at a higher level (e.g. https://github.com/apache/beam/blob/f0e0c2fc8a9a74bb051c922ef0f42c3ff494ee65/sdks/python/apache_beam/ml/transforms/base.py#L87)?